mods/ra/maps/sarin-gas-3-controlled-burn/controlledburn.lua

```diff
@@ -99,6 +107,28 @@ PowerDownTeslas = function()
 	end
 end
 
+PrepareFinishingHunt = function(player)
+	local buildings = GetBaseBuildings(player)
+
+	Trigger.OnAllKilledOrCaptured(buildings, function()
+		Utils.Do(player.GetGroundAttackers(), IdleHunt)
+
+		local harvesters = player.GetActorsByType("harv")
+		Utils.Do(harvesters, function(harvester)
+			-- Skip idle trucks for USSR's required units check.
+			harvester.Owner = BadGuy
+		end)
+	end)
+end
```

Comment:
```suggestion
PrepareFinishingHunt = function(player)
	local buildings = GetBaseBuildings(player)
	Trigger.OnAllKilledOrCaptured(buildings, function()
		Utils.Do(player.GetGroundAttackers(), function (actor)
			if (actor.Type == "harv") then
				-- Skip idle trucks for USSR's required units check.
				actor.Owner = BadGuy
			else
				IdleHunt(actor);
			end
		end )
	end)
end
```

Wouldn't something like this be better? Also why IdleHunt and not just Hunt?

Reply:
`GetGroundAttackers()` returns actors with AttackBase, which harvesters lack. The plain Hunt is a dandy shout though, since it wouldn't return dead actors.